File: doc/api/java/HelloWorld.java
import java.util.*;
import org.hyperdex.client.*;

public class HelloWorld
{
    public static void main(String[] args) throws HyperDexClientException
    {
        Client c = new Client("127.0.0.1", 1982);
        /* put */
        Map<String, Object> attrs = new HashMap<String, Object>();
        attrs.put("v", "Hello World!");
        System.out.println("put: " + c.put("kv", "k", attrs));
        /* get */
        System.out.println("got: " + c.get("kv", "k"));
    }
}

File: doc/ruby/client/hello-world.rb
require "hyperdex"
c = HyperDex::Client::Client.new("127.0.0.1", 1982)
c.put(:kv, "some key", {:v => "Hello World!"})
puts 'put "Hello World!"'
puts 'got:', c.get(:kv, "some key")
